common/src/main/resources/setup/classifications/ubogenre.xml

```diff
@@ -1,31 +1,36 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<mycoreclass ID="ubogenre" xmlns:xlink="http://www.w3.org/1999/xlink"
-             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-             xsi:noNamespaceSchemaLocation="MCRClassification.xsd">
+<mycoreclass ID="ubogenre" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="MCRClassification.xsd">
   <label xml:lang="en" text="Genre" description="Publication types of university bibliography"/>
   <label xml:lang="de" text="Genre" description="Publikationstypen der Hochschulbibliographie"/>
+
   <categories>
     <category ID="contribution">
       <label xml:lang="x-group" text="true"/>
       <label xml:lang="x-marcgt" text="book"/>
       <label xml:lang="x-mapping" text="marcgt:book orcidWorkType:journal-article"/>
+      <label xml:lang="x-mapping-his-genre" text="default:Sonstiger Publikationstyp"/>
+
       <label xml:lang="de" text="Artikel/Beiträge"/>
       <label xml:lang="en" text="Article/Contribution"/>
 
       <category ID="article">
         <label xml:lang="x-hosts" text="journal collection book newspaper video others"/>
         <label xml:lang="x-color" text="#7dabd0"/>
-        <label xml:lang="x-marcgt" text="article"/>
         <label xml:lang="x-mapping" text="marcgt:article orcidWorkType:journal-article"/>
-        <label xml:lang="de" text="Artikel/Aufsatz"
-               description="Ein Artikel oder ein redaktioneller Teil veröffentlicht in einer wissenschaftlichen Zeitschrift"/>
+        <label xml:lang="x-mapping-his-genre" text="journal:Journalartikel, newspaper:Journalartikel,  default:Sonstiger Publikationstyp"/>
```

Comment:
listings in label/@text should be seperated with only one whitspace (no comma etc.); see x-hosts or x-mapping

Reply:
True, but in this case I do need another separator since there are mappings containing blanks. This will be removed anyway once the mycore classfication mapping is in place.